features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ctxspend/dialogs/GiftCardDetailsViewModel.kt

```diff
@@ -161,10 +162,29 @@ class GiftCardDetailsViewModel @Inject constructor(
                                 if (!giftCard.barcodeUrl.isNullOrEmpty()) {
                                     saveBarcode(giftCard.barcodeUrl)
                                 }
+                            } else if (!giftCard.redeemUrl.isNullOrEmpty()) {
+                                log.error("CTXSpend returned a redeem url card: not supported")
+                                error.postValue(
+                                    CTXSpendException(
+                                        ResourceString(
+                                            R.string.gift_card_redeem_url_not_supported,
+                                            listOf(giftCard.id, giftCard.paymentId, txid)
+                                        )
+                                    )
+                                )
```

Comment:
For redeem url merchants, we will show an error.  Eventually CTX will label all of these properly so we can ignore them with our queries.